jwst/pipeline/calwebb_spec2.py

```diff
@@ -290,29 +288,42 @@ def process_exposure_product(
         if exp_type in ['NRS_FIXEDSLIT', 'NRS_MSASPEC', 'MIR_LRS-FIXEDSLIT'] \
            and not isinstance(calibrated, datamodels.CubeModel):
 
-            # Call the resample_spec step for 2D slit data
+            # Call pixel replace, followed by resample_spec  for 2D slit data
             resampled = calibrated.copy()
+            self.log.info("calling pixel replace NRS FS, MSA, LRS FIXED SLIT")
```

Comment:
Not sure this log message is really necessary for users. The step itself will automatically produce a log message saying that it's running. If you're just putting this in to verify which IF-block is being traversed in the code, then it would be more appropriate to change this to a debug message, rather than info.